contracts/Rivendale.sol

```diff
@@ -111,19 +126,25 @@ contract Rivendale {
         );
         require(!voteBreakdown[_id].executed, "vote has already been settled");
         if (voteBreakdown[_id].tally > 500) {
+            // Should we allow votes with value?
             (succ, res) = destination.call(data); //can we call this contract?
         }
         voteBreakdown[_id].executed = true;
         emit VoteSettled(_id, voteBreakdown[_id].tally > 500);
     }
 
+    // Similar to the OZ zeppelin issues found in the core contracts, we should consider higher limitations for voting
+    // 1.Disallow to vote on future issues
+    // 2. Disallow to vote on settled issues
     function vote(uint256 _id, bool _supports) external {
         require(!voted[msg.sender][_id], "address has already voted");
         require(voteBreakdown[_id].startDate > 0, "vote must be started");
         //Inherit Fellowship
         Fellowship _fellowship = Fellowship(fellowship);
         uint256[3] memory weightedVotes;
         //If the sender is a supported Walker (voter)
+
+        //walkers with pending withdraws aren't allowed to vote.Should they?

```

Comment:
yeah, you wouldn't want someone who gets to leave tomorrow to vote on a scenario that takes a week to settle